assets/css/stripeStyle.css

```diff
@@ -232,7 +232,7 @@ table tr {
 	margin: 0 -0.5rem;
 }
 
-.button-bar button {
+.button-bar button, .coupon-button {
```

Comment:
Codacy found an issue: [Expected selector ".button-bar button" to come before selector "button:hover:enabled" (no-descending-specificity)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5954708)